third_party/rust/Cargo.toml

```diff
@@ -55,8 +55,8 @@ raw_tty = "0.1.0"
 regex = "1.7"
 rsa = "0.8"
 rusb = "0.8.1"
-serde = { version="1.0", features=["serde_derive"] }
-serde_derive = "1.0"
+# serde-derive 1.0.172 uses precompiled binary which does not work well with Bazel.
+serde = { version="=1.0.171", features = ["derive"] }
```

Comment:
Can you create an issue in lowRISC/opentitan for this and leave a TODO comment here:
```
# TODO(opentitan#xxxxx): serde-derive 1.0.172 uses ...
```